vpc-cockroachdb-mzr/variables.tf

```diff
@@ -70,15 +70,28 @@ variable "vpc_database_image_name" {
 
 variable "cockroachdb_binary_url" {
   description = "The url for the cockroacdb download."
-  default = "https://binaries.cockroachdb.com"
+  default     = "https://binaries.cockroachdb.com"
 }
 
 variable "cockroachdb_binary_archive" {
   description = "The archive filename for the cockroacdb download."
-  default = "cockroach-v20.2.4.linux-amd64.tgz"
+  default     = "cockroach-v20.2.4.linux-amd64.tgz"
 }
 
 variable "cockroachdb_binary_directory" {
   description = "The directory for the cockroacdb archive when extracted."
-  default = "cockroach-v20.2.4.linux-amd64"
-}
\ No newline at end of file
+  default     = "cockroach-v20.2.4.linux-amd64"
+}
+
+variable "create_secrets_manager_instance" {
+  description = "Indicates whether or not to create a Secrets Manager instance, values can be true or false, default to false."
+  default     = false
+}
+
+variable "secrets_manager_instance_name" {
```

Comment:
@dprosper 

I guess this should have a default empty value as if create == true you will not use the name but without a default, you would still require it

Reply:
I actually no longer need the name, I am making it based on the prefix